teradatalabs/mapr52-hive/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Copyright 2017 Teradata
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM teradatalabs/mapr52-base
+MAINTAINER Teradata Docker Team <docker@teradata.com>
+
+# ADD ALL REQUIRED SCRIPTS AND FILES TO ROOT DIRECTORY
+ADD files/*.sh /root/
+ADD files/conf/hive-site.xml /opt/mapr/hive/hive-1.2/conf/hive-site.xml
+ADD files/conf/core-site.xml /opt/mapr/hadoop/hadoop-2.7.0/etc/hadoop/core-site.xml
+ADD files/supervisord.conf /etc/supervisord.conf
+COPY files/supervisord.d/* /etc/supervisord.d/
+
+RUN chmod 777 /root/*.sh \
+# INSTALL UTILITY SOFTWARE  
```

Comment:
don't install utility software, unless it's in a centos6-* image and we've all agreed it's needed